backend/src/main/java/edu/pl/masi/yellow/entity/UserEntity.java

```diff
@@ -0,0 +1,61 @@
+package edu.pl.masi.yellow.entity;
+
+import javax.persistence.*;
+
+@Entity
+@Table(name="user_data")
+public class UserEntity {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private int id;
+
+    @Column(name="user_name")
+    private String username;
+
+    @Column(name="user_pass")
+    private String password;
+
+    @Column(name="user_role")
+    private String role;
```

Comment:
Maybe it will be nice to define enum with predefined roles instead of String value?

Reply:
enum part with automatic serialization code is kind of tricky :/